README.md

```diff
@@ -31,4 +31,4 @@
 
 #### UML
 
-Link to an image of the UML for your application and response to events
+- ![uml](/images/auth-api-server-uml.jpg)
```

Comment:
The link to add the UML isn't working, I think that is because the file has .JPG and you are looking for .jpg but I am not completely certain